package.json

```diff
@@ -66,6 +68,8 @@
     "jquery-ui-npm": "^1.12.0",
     "lru_map": "^0.3.3",
     "marked": "^4.0.17",
+    "marked": "^4.0.10",
```

Comment:
please omit as it's updated to 4.0.17

Reply:
yeah i missed that, i already deleted that line.